README.md

```diff
@@ -1,41 +1,112 @@
-# Zero Forcing Number
-This program calculates the zero forcing number (and set) of any graph. The zero forcing number is defined as follows:
->A subset S of initially infected vertices of a graph G is called forcing if we can infect the entire graph by iteratively applying the following process. At each step, any infected vertex which has a unique uninfected neighbour, infects this neighbour. The forcing number of G is the minimum cardinality of a forcing set in G.
+# zftools
 
-\- [Kalinowski, Kamčev, Sudakov](https://arxiv.org/abs/1705.10391)
+### Purpose.
+
+This software provides very fast calculation of zero forcing parameters.  Currently, it is able to compute only the “original” zero forcing parameter, namely the **zero forcing number of a finite simple graph**, originally defined by the “Special Graphs Working Group” that formed during a 2006 workshop at the American Institute of Mathematics (AIM) in their paper:
+
+<ul>
```

Comment:
Do we want to use Markdown here (a simple `-`) instead? Otherwise I believe [you'd need](https://www.w3schools.com/html/html_lists_unordered.asp) a `<li>` on the line below to get the same desired output, if I'm understanding the intention correctly.

Reply:
I know, right?  I got this from https://stackoverflow.com/a/61500699.  There are other answers there that offer alternative options, but compared with those other possibilities, but this seemed like the least bad one.  I'm definitely open to other ways to get the same behavior if there are any!

Reply:
Ahh okay you were only looking for the indent. In that case, this is fine, disregard!